PopPUNK/refine.py

```diff
@@ -220,10 +211,10 @@ def refineFit(distMat, sample_names, start_s, mean0, mean1,
     if not no_local:
         sys.stderr.write("Trying to optimise score locally\n")
         local_s = scipy.optimize.minimize_scalar(newNetwork,
-                        bounds=bounds,
-                        method='Bounded', options={'disp': True},
+                        bounds = bounds,
```

Comment:
No space is the correct way in python (space is correct in R, unhelpfully)

Reply:
Never thought I'd say it, but R seems sensible.

Reply:
We don't run PEP on this (and I don't really care either), but just to avoid changing things!

NB: I will definitely get this wrong in future

Reply:
Noted - not sure why this particular example annoyed me, but for once I was a fan of whitespace.